marathon/app.go

```diff
@@ -24,6 +29,38 @@ type App struct {
 	VersionInfo VersionInfo       `json:"versionInfo"`
 }
 
+// isExcused check if application behavior is tolerated without consequence
+func (app App) isExcused() bool {
```

Comment:
Why you change immuinyt to excuse? I'll prefer to name it `hasImmunity` just to be compatible with documentation and code.